config/components/webhook/manifests.yaml

```diff
@@ -20,7 +20,6 @@ webhooks:
     - v1alpha2
     operations:
     - CREATE
```

Comment:
We don't have any way to automate the chart so can you make sure these changes are on the chart also?